roles/network_plugin/calico/defaults/main.yml

```diff
@@ -7,7 +7,7 @@ calico_pool_name: "default-pool"
 calico_ipv4pool_ipip: "Off"
 
 # Use IP-over-IP encapsulation across hosts
-ipip: true
+ipip: false
```

Comment:
your disable IPIP tunneling in Calico, and VXLAN tunneling also disabled.

In what mode will Calico build tunnels? )

Reply:
When I use Calico on Kubespray (mainly onpremise and Azure vms for testing), I set
```
calico_network_backend: vxlan
calico_vxlan_mode: Always
calico_ipip_mode: Never
```
because of https://docs.projectcalico.org/reference/public-cloud/azure
line 13 can be discussed because `calico_vxlan_mode: Never` doesn't work the above Azure case.
Anyways I never see any cases I needed to enable ipip encapsulation, I am fine to enable vxlan by default.

Reply:
@champtar was suggesting to enable vxlan only for "CrossSubnet" which to me makes more sense and seems to be the safest option while not forcing encapsulation where nodes have direct layer 2 connectivity. In my case I use kubernetes in openstack and on baremetal with direct connectivity so I disable any encapsulation.

FYI, the bug with ipip that triggered this work has been fixed in calico upstream and is currently tagged to land in 3.24.x.

/cc @champtar @floryut what do you think?